<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #2193b0,
        stop: 0.5 #3498db,
        stop: 1 #6dd5ed
    );
}</string>
  </property>
  <widget class="QWidget" name="wdLOGIN" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#wdLOGIN {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(52, 73, 94, 230),
        stop: 1 rgba(52, 73, 94, 200)
    );
    border-radius: 15px;
    border: 2px solid rgba(255, 255, 255, 0.1);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_108" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_108{
border:none;

background-color:transparent;
border-right: 1px solid rgb(157, 157, 157);}</string>
      </property>
      <widget class="QLabel" name="lblLogin">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>321</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>8-bit Arcade In</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 255, 255);
border:none;

background-color:transparent;
</string>
       </property>
       <property name="text">
        <string>COFFEE MTU</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wdLogin" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>350</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#wdLogin {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 rgba(255, 255, 255, 0.95),
        stop: 1 rgba(240, 255, 255, 0.9)
    );
    border-radius: 15px;
    border: 1px solid rgba(255, 255, 255, 0.3);
}</string>
      </property>
      <widget class="QLineEdit" name="txtUser">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#txtUser {
	color: rgb(65, 66, 63);
    background-color: transparent;
	border:none;
	font: 500 20pt &quot;8-bit Arcade In&quot;;
    padding top :10px;
	text-align: center;
	  border-bottom: 1px solid rgb(179, 179, 179)
}

QLineEdit#txtUser:focus {
    border-color: 1px solid rgb(172,12,32);
}

QLineEdit#txtUser:hover {
     border-bottom: 1px solid rgb(9, 117, 167);
}

QLineEdit:disabled {
    background-color: rgba(226, 226, 226, 1);
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPass">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#txtPass {
	color: rgb(65, 66, 63);
    background-color: transparent;
	border:none;
	font: 500 10pt &quot;8-bit Arcade In&quot;;
    padding top :10px;
	text-align: center;
	  border-bottom: 1px solid rgb(179, 179, 179)
}

QLineEdit#txtPass:focus {
    border-bottom: 1px solid rgb(9, 117, 167);
}

QLineEdit#txtPass:hover {
     border-bottom: 1px solid rgb(9, 117, 167);
}

QLineEdit:disabled {
    background-color: rgba(226, 226, 226, 1);
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>************</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLogin_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>240</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>8-bit Arcade In</family>
         <pointsize>23</pointsize>
         <italic>false</italic>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #2980b9,
        stop: 1 #3498db
    );
    font: 500 23pt &quot;8-bit Arcade In&quot;;
    border-radius: 5px;
    color: white;
}
QPushButton:hover {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #27ae60,
        stop: 1 #2ecc71
    );
    color: white;
}
QPushButton:pressed {
    background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 0,
        stop: 0 #219a52,
        stop: 1 #27ae60
    );
}</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
